src/test/java/org/apache/sling/event/impl/jobs/config/JobManagerConfigurationTestFactory.java

```diff
@@ -52,7 +52,10 @@ public long startup_delay() {
             public int cleanup_period() {
                 return 0;
             }
-            
+
+            public int progresslog_maxCount() {
+                return 0;
```

Comment:
IIUC this now changes the behaviour of log() for tests, as `0` indicates "discard logs". Was that the intention?

Reply:
Yes, this was intentional to set disable it for tests and write log specific unit cases to test the functionality.

Reply:
I was wondering if that was the right default for tests though, I would have assumed that for tests (that don't focus purely on log) you'd want the previous behaviour of unlimited logging. 
(Have checked for this specific test factory and it's only used by 1 class currently which doesn't use log - so this change has no influence for now)

Reply:
Addressed this review comment in [29f6d82](https://github.com/apache/sling-org-apache-sling-event/pull/24/commits/29f6d82ffe8071d17d0298553a281beca8cdad7a)